Userland/Libraries/LibC/unistd.cpp

```diff
@@ -925,10 +926,24 @@ int gettid()
     return cached_tid;
 }
 
-int sysbeep(int tone)
+int sysbeep(size_t tone)
 {
-    int rc = syscall(SC_beep, tone);
-    __RETURN_WITH_ERRNO(rc, rc, -1);
+    int fd = open("/dev/beep", O_RDWR);
+    if (fd < 0)
+        return fd;
+    int rc = 0;
+    int saved_errno = 0;
+    if (tone > UINT16_MAX) {
+        saved_errno = -EINVAL;
+    } else {
+        ToneGenerationPacket packet { static_cast<u16>(tone) };
```

Comment:
Why are we making this an `u16` now? I guess it makes sense given the range enforced by the kernel, but shouldn't we just make the argument a `u16` then as well?